huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -522,6 +523,29 @@ func resourceEvsVolumeUpdate(ctx context.Context, d *schema.ResourceData, meta i
 		}
 	}
 
+	if d.HasChanges("iops", "throughput") {
```

Comment:
The fields `iops` and `throughput` is **ForceNew** now in schema. Same thing in the documentation.